### PR TITLE
feat: #22: animation of arrow in hero component

### LINKED ISSUE #22: Build hero arrow animation
## Expected Behavior
Create the bouncing movement of the arrow from the hero component

### PR DESCRIPTION
**Pull request by Jaimeandres97**
## Description

- Create the animation of the arrow in the hero component.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [X] `npm run dev`

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by juancho11gm — APPROVED** [review]

**Comment by juancho11gm** [comment]
Please fix your commit message, there is a typo

**Review by juancho11gm — CHANGES_REQUESTED** [review]
I approved this PR by mistake

**Comment by juancho11gm** [comment]
You are missing the hover state

File: src/components/Hero/Hero.scss [hunk 1]
```diff
@@ -130,11 +130,31 @@
...
+    40% {
+      transform: translateY(-30px);
+      transform: translate(-50%, -7px);
+    }
+
+    60% {
+      transform: translateY(-15px);
+      transform: translate(-50%, -3.5px);
+    }
+  }
```
Comment (juancho11gm) [inline src/components/Hero/Hero.scss]:
```suggestion
 @keyframes bounce {
    0%,
    20%,
    50%,
    80%,
    100% {
      transform: translate(-50%, 0);
    }

    40% {
      transform: translate(-50%, -7px);
    }

    60% {
      transform: translate(-50%, -3.5px);
    }
  }
```

Comment (Jaimeandres97) [reply src/components/Hero/Hero.scss]: Sorry, I didn´t delete it from a previous test

**Review by juancho11gm — APPROVED** [review]

**Pull request merged**